fhir-server-test/src/test/java/com/ibm/fhir/server/test/bulkdata/ExportOperationTest.java

```diff
@@ -55,19 +56,29 @@
     public static final String BASE_VALID_URL = "/$export";
     public static final String BASE_VALID_STATUS_URL = "/$bulkdata-status";
     public static final String FORMAT = "application/fhir+ndjson";
+    private final String tenantName = "default";
+    private final String dataStoreId = "default";
 
     // Disabled by default
     private static boolean ON = false;
+    private static boolean isUseMinio = false;
+    private static boolean isUseMinioInBuildPipeline = false;
 
     public static final boolean DEBUG = false;
     private String exportStatusUrl;
     private String savedPatientId, savedPatientId2;
     private String savedGroupId, savedGroupId2;
-
+    private String minioUserName;
+    private String minioPassword;
+  
     @BeforeClass
     public void setup() throws Exception {
         Properties testProperties = TestUtil.readTestProperties("test.properties");
         ON = Boolean.parseBoolean(testProperties.getProperty("test.bulkdata.export.enabled", "false"));
+        isUseMinio = Boolean.parseBoolean(testProperties.getProperty("test.bulkdata.useminio", "false"));
```

Comment:
I'm still not sure how I feel about expanding our use of test.properties.  That properties file was originally just the client properties (used to instantiate FHIRClient).